pkg/operator/ceph/csi/secrets.go

```diff
@@ -88,7 +88,7 @@ func cephCSIKeyringRBDNodeCaps() []string {
 
 func cephCSIKeyringRBDProvisionerCaps() []string {
 	return []string{
-		"mon", "profile rbd",
+		"mon", "profile rbd, allow command 'osd blocklist'",
```

Comment:
same change is required for external python script also (if not present)

Reply:
it's present